cardano-cli/CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## 8.1.0
 
--
+- Updated ledger dependency bounds
+- Support for query flag in `Ping` parser
```

Comment:
Where can I find more documentation regarding this?

Reply:
Perhaps @coot or @jprider63 knows?

Reply:
Here is the [corresponding issue](https://github.com/input-output-hk/ouroboros-network/issues/3907) and [initial PR](https://github.com/input-output-hk/ouroboros-network/pull/4256).

Reply:
Thanks. Can you update the CHANGELOG following the format of the other entries?

Reply:
ok, I changed them, do they look okay now?